### DISCUSSION TITLE
Selected menu bar of all the tabs (`Code``Issues`...) of project are fixed at PR, no toggling

### DISCUSSION THREAD

**Discussion by astraw99** [community/community#31895] [Projects and Issues]
When toggling between the tabs of the project, it just fixed at PR tab.
For example, we click the `Issues` tab, but the selected menu bar just pointed to the PR tab.

URL: https://github.com/kubernetes/kubernetes/issues
<img width="913" alt="GitHub-issue-tabs" src="https://user-images.githubusercontent.com/25424734/188295308-fcacda5d-ea3e-4c18-99c8-ec3543ff386f.png">

The other 6 tabs are all like this. PTAL thanks.

**Comment by jfullstackdev** [comment]
yup, it's happening even in my own repos, if you will reload it, that's the time the highlight will move, but still the page is loading, a frontend issue
